### PR TITLE
Migration DMP Tool MUG version 4

### PR DESCRIPTION
**Pull request by lpandath**
## Description
Migration DMP Tool MUG version 4

### Checks

- [ ] Summary updated
- [ ] Version view updated
- [ ] Documentation added
- [ ] Tests added
- [ ] E2e tests created
- [ ] Successfully ran e2e tests before merge

closes GH-

### CODE REVIEW

**Comment by lpandath** [comment]
@rekt-hard I  would like to wait before merging to get feedback from Therese or Anne; other than that, it should be ready.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -2,11 +2,18 @@ name: release
...
   REGISTRY: ghcr.io
+  is-master-branch: ${{ github.ref == 'refs/heads/master' }}
+  is-next-branch: ${{ github.ref == 'refs/heads/next' }}
+  is-version-tag: ${{ startsWith(github.ref, 'refs/tags/v') }}
 
 on:
   push:
+    branches:
+      - master
+      - next
```
Comment (rekt-hard) [inline .github/workflows/release.yml]: Can be removed. We only want to run this workflow when we push tags.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -2,11 +2,18 @@ name: release
...
     tags:
       - v*
+  workflow_dispatch:
```
Comment (rekt-hard) [inline .github/workflows/release.yml]: Can be removed. We only want to run this workflow when we push tags.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -31,6 +38,11 @@ jobs:
         uses: docker/metadata-action@v5
         with:
           images: ${{ env.REGISTRY }}/${{ github.repository }}
+          # git tag: v1.2.3, output: 1.2.3
+          tags: |
+            type=semver,pattern={{version}},enable=${{ env.is-version-tag == 'true'}}
+            type=raw,value={{branch}},enable=${{ env.is-master-branch == 'true' || env.is-next-branch == 'true' }}
```
Comment (rekt-hard) [inline .github/workflows/release.yml]: Can be removed, since this will only run on tag pushes

**Review by rekt-hard — COMMENTED** [review]
In section 3, the button text is not visible
image

There seems to be an issue with the dropdown of pre-defined answers.
For instance, in TU Graz instance we get a dropdown like this:
image

This does not work properly with this PR:
image

**Comment by lpandath** [comment]
> In section 3, the button text is not visible image
> 
> There seems to be an issue with the dropdown of pre-defined answers. For instance, in TU Graz instance we get a dropdown like this: image
> 
> This does not work properly with this PR: image

So the drop-down is out, because this is how MUG required in there sheet and we already had an discussion for this part before. This was already in the last release. In relation with button color i will fix that. 

Here an screenshot that we agreed on the decisions:
Screenshot 2024-07-25 at 10 01 52

File: .github/workflows/release.yml (hunk 3 above)
Comment (rekt-hard) [reply .github/workflows/release.yml]: please keep lines 41-43. Only 44 should be removed

**Comment by rekt-hard** [comment]
> > In section 3, the button text is not visible image
> > There seems to be an issue with the dropdown of pre-defined answers. For instance, in TU Graz instance we get a dropdown like this: image
> > This does not work properly with this PR: image
> 
> So the drop-down is out, because this is how MUG required in there sheet and we already had an discussion for this part before. This was already in the last release. In relation with button color i will fix that.
> 
> Here an screenshot that we agreed on the decisions: Screenshot 2024-07-25 at 10 01 52

Ah right, missed that. Thank you for pointing it out

File: .github/workflows/release.yml (hunk 3 above)
Comment (lpandath) [reply .github/workflows/release.yml]: Sure, because here were all commented.

**Review by rekt-hard — APPROVED** [review]
good job!

File: apps/damap-frontend/src/app/components/layout/layout.component.html [hunk 4]
```diff
@@ -100,10 +97,19 @@
...
-    }}</a>
+    <div>
+      {{ "layout.version" | translate }}: {{ version }}
+      <a [routerLink]="'/gdpr'" rel="noopener">{{
+        "layout.gdpr" | translate
+      }}</a>
+    </div>
+    <a
+      class="mat-focus-indicator"
+      [href]="'layout.baseLogo.link' | translate"
```
Comment (rekt-hard) [inline apps/damap-frontend/src/app/components/layout/layout.component.html:108]: The translation string for the link is missing

**Pull request merged**